.github/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
 
 ## 0.13.0
-
+1. [A380X/PERF] Fixed issue where 0 was not allowed as cost index - @Jonny23787 (Jonny_23)
```

Comment:
Please put new changes at the end of the 0.13.0 list, and also don’t remove the blank lines around headings.